src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -43,239 +44,244 @@ public class Shooter extends SubsystemBase {
+	 *
+	 * @return the top motor's RPM
+	 * @see getBottomMotorRPM
+	 * @see getTopMotorSpeed
+	 * @see atSetpoint
+	 */
 	public double getTopMotorRPM() {
-		// REV does not let us set the encoder velocity in a simulation, so use simulated value
+		// REV does not let us set the encoder velocity in a simulation, so use
+		// simulated value
 		if (RobotBase.isSimulation()) {
-			return topMotorSpeedSim;
+			return leftMotorSpeedSim;
 		} else {
-			return topMotorEncoder.getVelocity();
+			return leftMotorEncoder.getVelocity();
 		}
 	}
 
 	/**
-	* Gets the RPM of the bottom motor, checking whether this is a simulation.
-	*
-	* @return the bottom motor's RPM
-	* @see    getTopMotorRPM
-	* @see    getBottomMotorSpeed
-	* @see    atSetpoint
-	*/
+	 * Gets the RPM of the bottom motor, checking whether this is a simulation.
+	 *
+	 * @return the bottom motor's RPM
+	 * @see getTopMotorRPM
+	 * @see getBottomMotorSpeed
+	 * @see atSetpoint
+	 */
 	public double getBottomMotorRPM() {
-		// REV does not let us set the encoder velocity in a simulation, so use simulated value
+		// REV does not let us set the encoder velocity in a simulation, so use
+		// simulated value
 		if (RobotBase.isSimulation()) {
-			return bottomMotorSpeedSim;
+			return rightMotorSpeedSim;
 		} else {
-			return bottomMotorEncoder.getVelocity();
+			return rightMotorEncoder.getVelocity();
 		}
 	}
 
 	/**
-	* Gets the speed at the surface of the top wheels.
-	*
-	* @return the top wheel's speed in m/s
-	* @see    getBottomMotorSpeed
-	* @see    getTopMotorRPM
-	* @see    atSetpoint
-	*/
+	 * Gets the speed at the surface of the top wheels.
+	 *
+	 * @return the top wheel's speed in m/s
+	 * @see getBottomMotorSpeed
+	 * @see getTopMotorRPM
+	 * @see atSetpoint
+	 */
 	public double getTopMotorSpeed() {
 		return shooterRPMToSpeed(getTopMotorRPM());
 	}
 
 	/**
-	* Gets the speed at the surface of the bottom wheels.
-	*
-	* @return the bottom wheel's speed in m/s
-	* @see    getTopMotorSpeed
-	* @see    getBottomMotorRPM
-	* @see    atSetpoint
-	*/
+	 * Gets the speed at the surface of the bottom wheels.
+	 *
+	 * @return the bottom wheel's speed in m/s
+	 * @see getTopMotorSpeed
+	 * @see getBottomMotorRPM
+	 * @see atSetpoint
+	 */
 	public double getBottomMotorSpeed() {
 		return shooterRPMToSpeed(getBottomMotorRPM());
 	}
-	
+
 	/**
-	* Gets the difference in RPM between the top motor and bottom motor.
-	* 
-	* @return the top motor's RPM - the bottom motor-s RPM
-	* @see    getMotorSpeedDifference
-	* @see    getTopMotorRPM
-	* @see    getBottomMotorRPM
-	* @see    atSetpoint
-	*/
+	 * Gets the difference in RPM between the top motor and bottom motor.
+	 * 
+	 * @return the top motor's RPM - the bottom motor-s RPM
+	 * @see getMotorSpeedDifference
+	 * @see getTopMotorRPM
+	 * @see getBottomMotorRPM
+	 * @see atSetpoint
+	 */
 	public double getMotorRPMDifference() {
 		return getTopMotorRPM() - getBottomMotorRPM();
```

Comment:
These methods still reference top/bottom motors.

Reply:
not anymore